lib/build/cpp/cxx-bootstrap

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
@iscottb122 I don't particularly love this approach, but it works and given that you're doing some reconfiguring of the sysroot at the musl level then it's possible that when we merge those changes in there will be a better way to handle this situation that's not available to me right now, so I've not put much effort into making this any nicer currently.